app/src/renderer/components/wallet/PageSend.vue

```diff
@@ -105,27 +105,25 @@ export default {
       await this.walletSend({
         fees: { denom, amount: 0 },
         to: address,
-        amount: [{ denom, amount }],
-        cb: (err) => {
-          this.sending = false
-          if (err) {
-            this.$store.commit('notifyError', {
-              title: 'Error Sending',
-              body: `An error occurred while trying to send: "${err.message}"`
-            })
-            return
-          }
-          this.$store.commit('notify', {
-            title: 'Successfully Sent',
-            body: `Successfully sent ${amount} ${denom.toUpperCase()} to ${address}`
-          })
+        amount: [{ denom, amount }]
+      }).then(() => {
+        this.sending = false
+        this.$store.commit('notify', {
+          title: 'Successfully Sent',
+          body: `Successfully sent ${amount} ${denom.toUpperCase()} to ${address}`
+        })
 
           // resets send transaction form
-          this.resetForm()
+        this.resetForm()
 
           // refreshes user transaction history
-          this.$store.dispatch('queryWalletHistory')
-        }
+        this.$store.dispatch('queryWalletHistory')
+      }, err => {
```

Comment:
is `err => {` the same as `.catch()`?

Reply:
Yes, but the `then(_, onReject)` will get called instead of `.catch`if available. See: https://codepen.io/faboweb/pen/WdmRoO

Reply:
cool! thanks for the example. still not sure when to use one vs the other...?